insights/parsers/yum_updates.py

```diff
@@ -0,0 +1,78 @@
+"""
+YumUpdates - parser for the `yum_updates` datasource
+====================================================
+Provides a list of available package updates, along with related advisories. This information
+is collected using DNF/YUM python interface.
+"""
+
+from insights import JSONParser, parser
+from insights.specs import Specs
+
+
+@parser(Specs.yum_updateinfo)
+class YumUpdates(JSONParser):
+    """
+
+    Expected output of the command is::
+
+    {
```

Comment:
@semtexzv this is causing the unexpected indentation error, it needs to be indented.

Reply:
Fixed